posthog/migrations/0291_create_person_override_model.py

```diff
@@ -0,0 +1,31 @@
+# Generated by Django 3.2.16 on 2023-01-18 10:29
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("posthog", "0290_add_dashboard_templates"),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name="PersonOverride",
+            fields=[
+                ("id", models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name="ID")),
```

Comment:
Should we use `BigAutoField` to not need to worry about running out of integer space? https://docs.djangoproject.com/en/4.1/ref/models/fields/#autofield

Reply:
Given that in the worst case we could have 1 override for all persons, and persons table could reach the int32 limit, this is a good idea :+1:. Will add a commit.

Reply:
should this be in the models class somehow too?